main.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass
```

Comment:
A more generic comment about the setup. I think it might be better to have a separate folder for frontend and backend in this repository to avoid a mix of python and js/ts config in the root. The turborepo/monorepo will still work when in a subfolder

Reply:
Created - as you suggested - a separet backend and frontend folder